test/e2e/service_options_test.go

```diff
@@ -49,6 +49,30 @@ func TestServiceOptions(t *testing.T) {
 	t.Run("delete hello service and returns no error", func(t *testing.T) {
 		test.serviceDelete(t, "hello")
 	})
+
+	t.Run("create hello service with min/max scale options and returns no error", func(t *testing.T) {
+		test.serviceCreateWithOptions(t, "hello", []string{"--min-scale", "1", "--max-scale", "3"})
+	})
+
+	t.Run("validate min scale value for service", func(t *testing.T) {
+		test.validateServiceMinScale(t, "hello", "1")
```

Comment:
I would add the validation step after the service call itself but in the same subtest, to make it not that fine granular.

Reply:
fixed it, thanks!